### DISCUSSION TITLE
Bottom to top text orientation

### DISCUSSION THREAD

**Discussion by leandrobattochio** [QuestPDF/QuestPDF#788] [Q&A]
How would I go to create a table with a vertical text such as following:

![image](https://github.com/QuestPDF/QuestPDF/assets/1385182/046c78aa-0adc-4fa6-baab-dca8d5f1f147)

I searched the documentation but I only found RTL, text left-to-right but that does not help. I'd need a bottom-to-top or top-to-bottom orientation. Is that possible?

**Comment by girlpunk** [comment]
RTL is specifically for languages that write in that style, this sounds like you want to [rotate](https://www.questpdf.com/api-reference/rotate.html) the text?

**Comment by leandrobattochio** [comment]
I tried but I can't reproduce. When I add more columns it goes crazy. I changed the layout so I don't have to use this, but if you think you can make shuch a table layout, please show me.
